examples/seismic/model.py

```diff
@@ -356,7 +356,7 @@ def _cfl_coeff(self):
         if 'lam' in self._physical_parameters or 'vs' in self._physical_parameters:
             coeffs = fd_w(1, range(-self.space_order//2+1, self.space_order//2+1), .5)
             c_fd = sum(np.abs(coeffs[-1][-1])) / 2
-            return np.sqrt(self.dim) / self.dim / c_fd
+            return .95 * np.sqrt(self.dim) / self.dim / c_fd
```

Comment:
so this justifies the change in expected norm values above?

Reply:
Yes. The CFL bound is extremely tight without it so for some complicated models it can become unstable, this is safer